app/models/authorization.rb

```diff
@@ -0,0 +1,35 @@
+# encoding: utf-8
+
+class Authorization
+  attr_accessor :token
+  attr_reader :token_auth_accounts
+
+  def initialize(token)
+    @token = token
+  end
+
+  def id
+    default_account.id
+  end
+
+  def default_account
```

Comment:
similarly, added this because AuthorizationRepresenter wants to return a default_account